<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrafTrajectory</class>
 <widget class="QWidget" name="GrafTrajectory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
